src/doom/g_game.c

```diff
@@ -2946,7 +2953,11 @@ void G_TimeDemo (char* name)
 boolean G_CheckDemoStatus (void) 
 { 
     int             endtime; 
-	 
+
+    // [crispy] catch the last cmd to track joins
+    ticcmd_t* cmd = last_cmd;
+    last_cmd = NULL;
```

Comment:
Doing this "swap" here because the code related to the join is under a couple branches but we want this to be reset whenever it's called.